app/assets/javascripts/discourse/controllers/composer.js.es6

```diff
@@ -109,6 +109,25 @@ export default Ember.Controller.extend({
             !creatingPrivateMessage;
   },
 
+  @computed('model.whisper', 'model.unlistTopic')
+  whisperOrUnlistTopic(whisper, unlistTopic) {
+    return whisper || unlistTopic;
+  },
+
+  @computed('model.whisper', 'model.unlistTopic')
+  whisperOrUnlistTopicText(whisper, unlistTopic) {
+    if (whisper) {
+      return I18n.t("composer.whisper");
+    } else if (unlistTopic) {
+      return I18n.t("composer.unlist");
+    }
+  },
+
+  @computed('model.creatingTopic')
+  canUnlistTopic(creatingTopic) {
```

Comment:
This can also be written as `canUnlistTopic: Ember.computed.alias("model.creatingTopic")`